clients/admin-ui/src/features/common/table/v2/FidesTable.tsx

```diff
@@ -137,6 +138,7 @@ const HeaderContent = <T,>({
         fontSize="xs"
         lineHeight={9} // same as table header height
         fontWeight="medium"
+        style={header.column.columnDef.meta?.headerProps as object}
```

Comment:
Prefer a more specific type for this:

```suggestion
        style={header.column.columnDef.meta?.headerProps as React.CSSProperties}
```